trade-finance-manager-ui/server/controllers/utilisation-reports/record-corrections/create-record-correction-request/validate-form-values.ts

```diff
@@ -0,0 +1,51 @@
+import { CreateRecordCorrectionRequestFormValues, CreateRecordCorrectionRequestErrorsViewModel, ErrorSummaryViewModel } from '../../../../types/view-models';
+
+/**
+ * The maximum length of the additional information field for a record correction request.
+ */
+export const MAX_RECORD_CORRECTION_ADDITIONAL_INFO_LENGTH = 500;
+
+/**
+ * Validates the additional information field for a record correction request.
+ * @param additionalInfo - The additional information text to validate.
+ * @returns An error message if validation fails, undefined otherwise.
+ */
+export const getAdditionalInfoValidationErrors = (additionalInfo: string | undefined): string | undefined => {
```

Comment:
It looks like this can only return one error at a time, so:
```suggestion
export const getAdditionalInfoValidationError = (additionalInfo: string | undefined): string | undefined => {
```